solidgpt/src/workskill/skills/llama_write_prd.py

```diff
@@ -0,0 +1,60 @@
+from solidgpt.src.manager.llamanager import LLAManager
+from solidgpt.src.manager.promptresource import PRODUCT_MANAGER_BRAINSTORM_OUTPUT_TEMPLATE, PRODUCT_MANAGER_BRAINSTORM_ROLE_ASSUMPTION, PRODUCT_MANAGER_PRD_OUTPUT_TEMPLATE, PRODUCT_MANAGER_PRD_ROLE_ASSUMPTION, build_gpt_prompt
+from solidgpt.src.util.util import *
+from solidgpt.src.workskill.workskill import *
+
+class WritePRD(WorkSkill):
```

Comment:
Please rename to WritePRDLlama etc to point out it is an llama2 skill

Reply:
Renamed to WritePRDLlama